ci/docker-compose.yml

```diff
@@ -5,19 +5,52 @@ services:
     build:
       context: .
       dockerfile: Dockerfile.validator
+    # uncomment to provide amount of bridge nodes to provision (default: 1)
+    # command: ["/opt/entrypoint.sh", "2"]
     volumes:
-      - shared:/shared
+      - credentials:/credentials
+      - genesis:/genesis
 
-  bridge:
+  bridge-0:
     image: bridge
     platform: "linux/amd64"
     build:
       context: .
       dockerfile: Dockerfile.bridge
-    volumes:
-      - shared:/shared
+    # uncomment to provide the id of the bridge node (default: 0)
+    # command: ["/opt/entrypoint.sh", "0"]
```

Comment:
I'm wondering whether providing node id with env variable would be less awkward?
Initially I wanted to suggest extracting it from container name, but there doesn't seem to be any good way to get to that from inside.

Reply:
let's see, switched to env